src/app/pages/signup/signup.page.ts

```diff
@@ -25,6 +25,6 @@ export class SignupPage implements OnInit {
 
   signUp() {
     let user: User = this.userForm.value;
```

Comment:
Perchè stai usando let? Questa variabile non viene riassegnata in seguito, pertanto usa il const.